.github/pull_request_template.md

```diff
@@ -14,7 +14,7 @@ Include the following checklist in your PR:
 
 ---
 
-**Template Β: This PR does not modify `cabal` behaviour (documentation, tests, refactoring, etc.)**
+**Template Β: This PR does not modify `cabal` behaviour or `Cabal` API (documentation, tests, refactoring, etc.)**
```

Comment:
I think this "or" should be "nor". To avoid the association of the parenthesised remark with "API", maybe add something, like "(so it's just a documentation ...". Though that starts to sound chatty. So maybe rather "API. Examples: documentation ...".

Reply:
"or" is fine; "nor" is primarily used with "neither".

Reply:
TIL